unleash-yggdrasil/src/strategy_parsing.rs

```diff
@@ -738,6 +759,13 @@ mod tests {
     #[test_case("user_id contains_any_ignore_case [\"EMAIL\"]", true)]
     #[test_case("user_id ends_with_any_ignore_case [\".COM\"]", true)]
     #[test_case("user_id starts_with_any_ignore_case [\"SOME\"]", true)]
+    #[test_case("user_id in_ignore_case [\"some-EMAIL.com\"]", true)]
+    #[test_case("user_id in_ignore_case [\"noemail.com\",\"neither-THIS.com\"]", false)]
+    #[test_case("user_id not_in_ignore_case [\"notemail.com\"]", true)]
+    #[test_case(
+        "user_id not_in_ignore_case [\"notemail.com\",\"some-EMAIL.com\"]",
+        false
+    )]
```

Comment:
Nice! These LGTM but I think we should have another set where the context value is also mixed case. These cover the case where

`"someuser" in_ignore_case ["sOMeUSer"]`

but it doesn't cover

`"sOMeUSer" in_ignore_case ["someuser"]`